python/rmm/_cuda/gpu.py

```diff
@@ -1,59 +1,36 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
-from rmm._cuda.gpu cimport (
-    CUresult,
-    cudaDeviceAttr,
-    cudaDeviceGetAttribute,
-    cudaDeviceProp,
-    cudaDriverGetVersion,
-    cudaError,
-    cudaError_t,
-    cudaGetDeviceCount,
-    cudaGetDeviceProperties,
-    cudaGetErrorName,
-    cudaGetErrorString,
-    cudaRuntimeGetVersion,
-    cuDeviceGetName,
-    cuGetErrorName,
-    cuGetErrorString,
-)
-
-from enum import IntEnum
+from cuda import cuda, cudart
 
 
 class CUDARuntimeError(RuntimeError):
-
-    def __init__(self, cudaError_t status):
+    def __init__(self, status: cuda.CUresult.CUDA_SUCCESS):
         self.status = status
-        cdef str name = cudaGetErrorName(status).decode()
-        cdef str msg = cudaGetErrorString(status).decode()
+        _, name = cudart.cudaGetErrorName(status)
+        _, msg = cudart.cudaGetErrorString(status)
```

Comment:
Shouldn't we be checking these error codes as we do below?

Reply:
Done!